_includes/tutorials/joining-stream-stream/ksql/markup/dev/create-orders-stream.adoc

```diff
@@ -0,0 +1,5 @@
+First, you'll need to create a Kafka stream and its underlying topic to represent the orders:
```

Comment:
We should call out that we're using an attribute of the event to control for time, and note that it'll be significant later.